examples/tcp_gps_example.py

```
@@ -15,32 +18,29 @@ def main() -> None:
                 return
 
             position = node.get("position")
             if position is None:
-                print("Node has no position data yet.", file=sys.stderr)
+                logger.error("Node has no position data yet.")
                 return
 
             print(position)
     except OSError as exc:
-        print(f"Could not connect to {RADIO_HOSTNAME}: {exc}", file=sys.stderr)
+        logger.exception("Could not connect to %s: %s", RADIO_HOSTNAME, exc)
```

Ruff (0.15.2)

[warning] Redundant exception object included in `logging.exception` call

(TRY401)